Composer/packages/client/src/pages/language-generation/index.tsx

```diff
@@ -202,13 +154,14 @@ const LGPage: React.FC<RouteComponentProps> = props => {
         </div>
         {lgFile && (
           <div css={contentEditor}>
-            {editMode ? (
-              <Suspense fallback={<LoadingSpinner />}>
-                <CodeEditor file={lgFile} template={inlineTemplate} line={line} />
-              </Suspense>
-            ) : (
-              <TableView file={lgFile} activeDialog={activeDialog} onClickEdit={onTableViewClickEdit} />
-            )}
+            <Suspense fallback={<LoadingSpinner />}>
+              <Router primary={false} component={Fragment}>
+                <CodeEditor path="edit" />
+                <CodeEditor path=":fileId/edit" />
```

Comment:
We probably just want to wrap the code editor paths with suspense. This may require moving Suspense into the CodeEditor component

Reply:
Yes, the only component need to be lazy loaded is `<LgEditor >` . but i meet some problem when i fellow those step
 
1. remove `Suspense` wrap in index.tsx, so lazy load shouldn't appear here.
```
// const CodeEditor = React.lazy(() => import('./code-editor'));
import CodeEditor from './code-editor';
```

2. add  `Suspense`  wrap in code-editor.tsx,` and lazy load `<LgEditor >`.
```
// import { LgEditor } from '@bfc/code-editor';
const { LgEditor } = React.lazy(() => import('@bfc/code-editor'));
```
this caused an import error,  after some digging, I found it's `React.lazy` haven't support named  module import. we have to create an intermediate module for reexporting.
I guess it cost more than the benefits.

Reply:
My comment doesn't make any sense anyways. We are still lazy importing the code editor route component.